doc/specs/stdlib_bitsets.md

```diff
@@ -194,8 +194,8 @@ undefined. These procedures are summarized in the following table:
 
 ### Assignments
 
-The module defines an assignment operation, `=`, that creates a
-duplicate of an original bitset. It also defines assignments to and
+The module uses the intrinsic assignment operation, `=`, to create a
+duplicate of an original bitset. It additionally defines assignments to and
```

Comment:
@degawa : should we add additional information based on the Discourse thread?

Reply:
Sorry for the late review. 

The additional information based on the Discourse thread is optional because the information, like below, is too broad for describing the specification of the bitset assignment:
The user-defined assignment operator causes the self-assignment problem with the argument aliasing; since the type `bitset_large` and `bitset_64` do not have any pointer component, the intrinsic assignment operation should be used.

Reply:
Thank you.

Reply:
@degawa Could you approve this PR, if ok for you, please? Then I will merge it.